res/values-ku/strings.xml

```diff
@@ -158,7 +158,7 @@
     <string name="Quadra_Gulp">"Şike çaran"</string>
     <string name="Multi_Gulp">"Zedeyi çaran"</string>
     <string name="Mega_Gulp">"Hemi çaran"</string>
-    <string name="Ultra_Gulp">"ultra çaran"</string>
+    <string name="Ultra_Gulp">"Ultra çaran"</string>
     <string name="Monster_Gulp">"Cenawir çaran"</string>
     <string name="Ludicrous_Gulp">"nerêkî di gulp"</string>
```

Comment:
I think this one should have the "N" capital too, if this is intended to be capital/small correction